cnd/src/storage/db/tables.rs

```diff
@@ -500,6 +447,47 @@ impl BtcDaiOrder {
 
         Ok(params)
     }
+
+    /// Move the amount that is settling from open to settling.
+    ///
+    /// Whilst we don't have partial order matching, this simply means updating
+    /// `settling` to the amount of `open` and updating `open` to `0`.
+    ///
+    /// Once we implement partial order matching, this will need to get more
+    /// sophisticated.
+    pub fn mark_as_settling(&self, conn: &SqliteConnection) -> Result<()> {
+        let affected_rows = diesel::update(self)
+            .set((
+                btc_dai_orders::settling.eq(Text::<Satoshis>(self.open.to_inner().into())),
+                btc_dai_orders::open.eq(Text::<Satoshis>(btc_zero().to_inner().into())),
+            ))
+            .execute(conn)?;
+
+        if affected_rows == 0 {
+            anyhow::bail!("failed to mark order {} as settling", self.order_id)
+        }
+
+        Ok(())
+    }
+
+    pub fn cancel(&self, conn: &SqliteConnection) -> Result<()> {
```

Comment:
```suggestion
    pub fn set_to_cancelled(&self, conn: &SqliteConnection) -> Result<()> {
```
It is best for the future dev to have some name consistency between method that act similarly.